core/src/com/unciv/logic/map/tile/TileHistory.kt

```diff
@@ -0,0 +1,64 @@
+package com.unciv.logic.map.tile
+
+import com.unciv.logic.IsPartOfGameInfoSerialization
+import java.util.*
+import kotlin.collections.HashMap
+
+/**
+ * Records events throughout the game related to a tile.
+ *
+ * Used for end of game replay.
+ */
+open class TileHistory : IsPartOfGameInfoSerialization {
+
+    class TileHistoryState : IsPartOfGameInfoSerialization {
```

Comment:
I think I had this initially and it complained about serialization. I think I was missing the default values since it complained about the lack of a default constructor. Just tried again and it works now with default values. Thanks for pointing this out. Done!